docs/docs/launch-manual/sections/core-hacking/sections/building-pulsar.md

```diff
@@ -10,8 +10,8 @@ The Pulsar application code can be found in the
 
 To build Pulsar you will need to meet some basic requirements:
 
-- Node.js 16 (recommended installation is via
-  [nvm](https://github.com/nvm-sh/nvm))
+- Node.js (version specified in [pulsar/.nvmrc](https://github.com/pulsar-edit/pulsar/blob/master/.nvmrc),
+  recommended installation is via [nvm](https://github.com/nvm-sh/nvm))
```

Comment:
```suggestion
  recommended installation is via [nvm](https://github.com/nvm-sh/nvm)) (For Windows User's [nvm-windows](https://github.com/coreybutler/nvm-windows) is recommended.)
```

Reply:
`User's` -> `users` but otherwise I think that is good to go.
Do you know if the Windows one fully supports .nvmrc on `nvm install` like the Linux one? The docs were a bit fuzzy, they implied it is supported with `nvm use` but don't explicitly mention it for `install`

Reply:
I'll have to go ahead and double check. I'm pretty sure I fully tested the first time making these recommendations, but I'll go ahead and confirm.